src/delta.f90

```diff
@@ -644,6 +658,77 @@ subroutine fill_tetrahedra_3d(tetra, evals, tetra_evals)
        end do
     end do
   end subroutine fill_tetrahedra_3d
+  
+  
+subroutine fermi_window_adjusted_fill_tetrahedra_3d(wann, crys, wave_vector_mesh, tetra, evals, tetra_evals)
```

Comment:
This name is not clear. Just calling it `fill_tetrahedra_3d_new` is fine. We will replace the old one with this following a demonstration of correctness.